app/src/main/res/layout/fragment_map.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".pages.map.MapFragment">
 
-    <!-- TODO: Update blank fragment layout -->
-    <TextView
+    <fragment
+        android:name="com.google.android.gms.maps.SupportMapFragment"
+        android:id="@+id/google_map"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:text="Map fragment" />
+        android:layout_height="match_parent"/>
```

Comment:
Just want to double check, it doesn't work if we put the `fragment_map_container` layout in here instead of instantiating a separate fragment, right?